knowledge/models/res_config.py

```diff
@@ -34,28 +15,3 @@ class KnowledgeConfigSettings(models.TransientModel):
         help='Document indexation, full text search of attachements.\n'
         '- This installs the module document.'
     )
-
-    module_document_page = fields.Boolean(
-        'Manage document pages (Wiki)',
-        help='Provide document page and category as a wiki.\n'
-        '- This installs the module document_page.'
-    )
-
-    module_document_page_approval = fields.Boolean(
-        'Manage documents approval',
-        help='Add workflow on documents per category.\n'
-        '- This installs the module document_page_approval.'
-    )
-
-    module_cmis_read = fields.Boolean(
-        'Attach files from an external DMS into Odoo',
-        help='Connect Odoo with a CMIS compatible server to attach files\n'
-        'to an Odoo record.\n'
-        '- This installs the module cmis_read.'
-    )
-
-    module_cmis_write = fields.Boolean(
-        'Store attachments in an external DMS instead of the Odoo Filestore',
-        help='Connect Odoo with a CMIS compatible server to store files.\n'
-        '- This installs the module cmis_write.'
-    )
```

Comment:
@LeartS Please keep those fields. Even if the modules are not migrated, it gives them visibility.